### ISSUE TITLE
Grid throws Error on Horizontal Scroll

### ISSUE THREAD

**Issue by slewis30328** [telerik/kendo-angular#4002]
**Describe the bug**

Upgraded recently to Angular 16 and the latest Kendo components. Started to see an error in browser console anytime grid began to scroll horizontally.  The following error is dumped for all app grid(s) when the scroll begins.  Have not seen any issues for grids that do not require a horizontal scroll. 

global-error-handler.ts:23 TypeError: Cannot read properties of undefined (reading 'offsetY')
    at RowReorderService.press (progress-kendo-angular-grid.mjs:6199:37)
    at GridComponent.handleReorderEvents (progress-kendo-angular-grid.mjs:19757:39)
    at GridComponent_ng_container_7_Template_kendo_grid_list_press_2_listener (progress-kendo-angular-grid.mjs:20689:26)
    at executeListenerWithErrorHandling (core.mjs:15659:16)
    at wrapListenerIn_markDirtyAndPreventDefault (core.mjs:15692:22)
    at platform-browser.mjs:666:17
    at platform-browser.mjs:1746:21
    at _ZoneDelegate.invoke (zone.js:368:26)
    at Object.onInvoke (core.mjs:26117:33)
    at _ZoneDelegate.invoke (zone.js:367:52)

**To Reproduce**

**Expected behavior**

**Screenshots**

**Browser**
 - OS: [for example, iOS]
 - Browser [for example, Chrome, Safari]
 - Version [for example, 22]

**Additional context**
{
  "name": "smart-nx",
  "version": "0.0.0",
  "license": "MIT",
  "scripts": {
    "start": "nx serve",
    "build": "nx build",
    "test": "nx test",
    "postinstall": ""
  },
  "private": true,
  "devDependencies": {
    "@angular-devkit/build-angular": "~16.0.1",
    "@angular-eslint/eslint-plugin": "~16.0.1",
    "@angular-eslint/eslint-plugin-template": "~16.0.1",
    "@angular-eslint/template-parser": "~16.0.1",
    "@angular/cli": "~16.0.1",
    "@angular/compiler-cli": "~16.0.1",
    "@angular/language-service": "~16.0.1",
    "@angular/localize": "~16.0.1",
    "@nx/angular": "^16.1.0",
    "@nx/cypress": "16.1.0",
    "@nx/eslint-plugin": "16.1.0",
    "@nx/express": "^16.1.0",
    "@nx/jest": "16.1.0",
    "@nx/js": "16.1.0",
    "@nx/linter": "16.1.01",
    "@nx/node": "^16.1.0",
    "@nx/workspace": "16.1.0",
    "@types/express": "4.17.17",
    "@types/jest": "^29.5.1",
    "@types/node": "20.1.7",
    "@typescript-eslint/eslint-plugin": "~5.59.6",
    "@typescript-eslint/parser": "~5.59.6",
    "cypress": "^12.12.0",
    "eslint": "~8.40.0",
    "eslint-config-prettier": "8.8.0",
    "eslint-plugin-cypress": "^2.13.3",
    "jest": "29.5.0",
    "jest-environment-jsdom": "29.5.0",
    "jest-preset-angular": "13.1.1",
    "ng-packagr": "^16.0.1",
    "postcss": "^8.4.23",
    "postcss-import": "^15.1.0",
    "postcss-preset-env": "^8.3.2",
    "postcss-url": "^10.1.3",
    "prettier": "^2.8.8",
    "supertest": "^6.3.3",
    "ts-jest": "^29.1.0",
    "ts-node": "10.9.1",
    "typescript": "~5.0.4"
  },
  "dependencies": {
    "@angular/animations": "~16.0.1",
    "@angular/cdk": "^16.0.1",
    "@angular/common": "~16.0.1",
    "@angular/compiler": "~16.0.1",
    "@angular/core": "~16.0.1",
    "@angular/forms": "~16.0.1",
    "@angular/platform-browser": "~16.0.1",
    "@angular/platform-browser-dynamic": "~16.0.1",
    "@angular/router": "~16.0.1",
    "@ckeditor/ckeditor5-angular": "^6.0.1",
    "@ckeditor/ckeditor5-build-decoupled-document": "^37.1.0",
    "@clarivate/angular-image-viewer": "^2.0.8",
    "@fortawesome/fontawesome-pro": "^6.4.0",
    "@ngxs/devtools-plugin": "^3.8.0",
    "@ngxs/logger-plugin": "^3.8.0",
    "@ngxs/storage-plugin": "^3.8.0",
    "@ngxs/store": "^3.8.0",
    "@progress/kendo-angular-buttons": "^12.0.1",
    "@progress/kendo-angular-charts": "^12.0.1",
    "@progress/kendo-angular-common": "^12.0.1",
    "@progress/kendo-angular-dateinputs": "^12.0.1",
    "@progress/kendo-angular-dialog": "^12.0.1",
    "@progress/kendo-angular-dropdowns": "^12.0.1",
    "@progress/kendo-angular-excel-export": "^12.0.1",
    "@progress/kendo-angular-gauges": "^12.0.1",
    "@progress/kendo-angular-grid": "^12.0.1",
    "@progress/kendo-angular-inputs": "^12.0.1",
    "@progress/kendo-angular-intl": "^12.0.1",
    "@progress/kendo-angular-l10n": "^12.0.1",
    "@progress/kendo-angular-label": "^12.0.1",
    "@progress/kendo-angular-layout": "^12.0.1",
    "@progress/kendo-angular-notification": "^12.0.1",
    "@progress/kendo-angular-pager": "^12.0.1",
    "@progress/kendo-angular-pdf-export": "^12.0.1",
    "@progress/kendo-angular-popup": "^12.0.1",
    "@progress/kendo-angular-progressbar": "^12.0.1",
    "@progress/kendo-angular-sortable": "^12.0.1",
    "@progress/kendo-angular-toolbar": "^12.0.1",
    "@progress/kendo-angular-tooltip": "^12.0.1",
    "@progress/kendo-angular-treeview": "^12.0.1",
    "@progress/kendo-angular-upload": "^12.0.1",
    "@progress/kendo-data-query": "^1.6.0",
    "@progress/kendo-drawing": "^1.17.5",
    "@progress/kendo-licensing": "^1.3.0",
    "@progress/kendo-theme-bootstrap": "^6.3.0",
    "@types/hammerjs": "^2.0.41",
    "adm-zip": "^0.5.10",
    "amqp-ts": "^1.8.0",
    "bootstrap": "^5.2.3",
    "canvas": "^2.11.2",
    "chartjs-to-image": "^1.2.1",
    "easy-pdf-merge": "^0.2.6",
    "exceljs": "^4.3.0",
    "express": "4.18.2",
    "express-fileupload": "^1.4.0",
    "hammerjs": "^2.0.8",
    "html-to-pdfmake": "^2.4.20",
    "jsbarcode": "^3.11.5",
    "ldapjs": "^3.0.2",
    "morgan": "^1.10.0",
    "nodemailer": "^6.9.2",
    "nodemailer-express-handlebars": "^6.1.0",
    "nodemailer-smtp-transport": "^2.4.2",
    "oracledb": "^5.5.0",
    "pdfmake": "^0.2.7",
    "rxjs": "~7.8.1",
    "tslib": "^2.5.0",
    "winston": "^3.8.2",
    "xss": "^1.0.14",
    "zone.js": "~0.13.0"
  }
}

![HorizScrollIssue](https://github.com/telerik/kendo-angular/assets/5197299/df77eecc-d343-4749-9139-5b36b3cce667)

**Comment by slavenai** [comment]
Hi @slewis30328 ,

Could you provide us with a runnable StackBlitz demo that demonstrates the described issue?
Since `v12.0.0` the `kendo-angular-utils` package is a peer dependency of `kendo-angular-grid` package. Please make sure the utils package is installed.

**Comment by slavenai** [comment]
Thank you for the detailed report. Based on the provided screenshot, I tried to reproduce the issue with similar Grid configuration and latest packages version, but to no avail:

https://stackblitz.com/edit/angular-9j9kvu-yucvkd?file=src%2Fapp%2Fapp.component.ts

Feel free to update the example above so that it demonstrates the reported issue and provide any additional information about the setup that will help us investigate further.

**Comment by slewis30328** [comment]
Figured out what was triggering the error in grid.  It's tied to the hammer.js module.  When I imported the hammer module into the app module - you will see the error.  We use the Kendo Charts - hence the need for hammer - according to your docs.

Edited StackBlitz:
https://stackblitz.com/edit/angular-9j9kvu-bn4fxk?file=src%2Fapp%2Fapp.module.ts

![image](https://github.com/telerik/kendo-angular/assets/5197299/f2fb187e-58e3-4d1f-951c-e9c79d5abb0f)
